### DISCUSSION TITLE
[Feature] View assets not in any album

### DISCUSSION THREAD

**Discussion by Sherlock1979** [immich-app/immich#2559] [Feature Request]
### The feature

It would be great to be able to view all assets that are not put in any albums yet.

It would help organise the photos. I would appreciate this feature a lot and it would help my categorisation and family photo sharing workflow.

Maybe it can be a search option or it could be accessed from the 'Library' section? We can call it maybe 'Uncategorised' 

(BTW Photoprism has it as a menu item within albums and calls it: 'unsorted' ?!)

Thanks for considering it.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1621
